packages/cloudflare/README.md

```diff
@@ -195,24 +195,42 @@ export default defineConfig({
 
 ### `runtime`
 
-`runtime: { mode: "off" | "local", persistTo: string }`
+`runtime: { mode: 'off' } | { mode: 'local'; type: 'pages'; persistTo?: string; bindings?: Record<string, CF_BINDING>} | { mode: 'local'; type: 'workers'; persistTo?: string; };`
 
 default `{ mode: 'off', persistTo: '' }`
 
 Determines whether and how the Cloudflare Runtime is added to `astro dev`.
+Read more about [the Cloudflare Runtime](#cloudflare-runtime).
 
-The Cloudflare Runtime includes [Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/bindings), [environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), and the [cf object](https://developers.cloudflare.com/workers/runtime-apis/request/#incomingrequestcfproperties). Read more about [accessing the Cloudflare Runtime](#cloudflare-runtime).
+The `type` property defines where your Astro project is deployed to:
 
-The `mode` property defines how the runtime is added to `astro dev`:
+- `pages`: Deployed to [Cloudflare Pages](https://pages.cloudflare.com/)
+- `workers`: Deployed to [Cloudflare Workers](https://workers.cloudflare.com/)
 
-- `local`: uses bindings mocking and locally static placeholders
-- `off`: no access to the Cloudflare runtime using `astro dev`. You can alternatively use [Preview with Wrangler](#preview-with-wrangler)
+The `mode` property defines what the runtime supports in `astro dev`:
 
-The `persistTo` property defines where the local runtime is persisted to when using `mode: local`. This value is a directory relative to your `astro dev` execution path.
+- `off`: no access to the runtime using `astro dev`. You can alternatively use [Preview with Wrangler](#preview-with-wrangler)
+- `local`: uses a local runtime powered by miniflare and workerd. This supports Cloudflare's Bindings currently. For unsupported features, you can alterntively use `moed: 'off'` in combination with [Preview with Wrangler](#preview-with-wrangler)
```

Comment:
```suggestion
- `local`: uses a local runtime powered by miniflare and workerd, which supports Cloudflare's Bindings. 
```
So, some problems understanding when I can/can't use this option that would ideally be addressed:

Forget for a moment that I might not know which features are unsupported (but, that's also a worry). Better to state when to use each one, rather than refer to the exact other configuration you already mentioned.

I'm also not sure, since you just said that Wrangler is an *alternative* to `off`, why I might now use `off` *with* Wrangler Preview.

Also, "Choose when" or "Only use if" kinds of statements are generally more helpful than saying "you can use the other configuration". Forget for a moment that in neither case do I know when or why I should/might choose Wrangler (though, I don't). I might prefer to add under `off` something about "use this for unsupported features such as...."